VideoIndexerEnabledByArc/scripts/install_aks_cluster.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+#===========================================================================================================#
+# @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ Helper Functions @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#===========================================================================================================#
+
+##############################################
+#  CLI Tools
+###############################################
+function install_cli_tools {
+  # https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli
+  echo "ensure you got the latest CLI client and install add ons if needed"
+  echo "https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli"
+  az extension add --name connectedk8s
+  az extension add --name k8s-extension
+  az extension add --name aks-preview
+  az provider register --namespace Microsoft.Kubernetes
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  az provider register --namespace Microsoft.ExtendedLocation
+
+  echo "install jq"
+  sudo apt-get update && apt-get install jq -y
```

Comment:
This assumes the customer runs it in ubuntu. We should document it in the README